configuration/pih/scripts/global/zl.js

```diff
@@ -0,0 +1,144 @@
+function setUpEdd(currentEncounterDate, msgWeeks) {
+  var updateEdd = function () {
+    var lastPeriodDate = htmlForm.getValueIfLegal("lastPeriodDate.value");
```

Comment:
Always use `const` or `let`, not `var`. `var` behaves in strange and surprising ways.

Reply:
```suggestion
    const lastPeriodDate = htmlForm.getValueIfLegal("lastPeriodDate.value");
```